content/docs/technical_practices/azure/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+# Page title as it appears in the navigation menu
+title: "Azure"
+# Adjust weight to reorder menu items (lower numbers appear first)
+weight: 6
+# Uncomment to hide nested pages in a collapsed menu
+# bookCollapseSection = true
+# Uncomment to hide this page from the navigation menu
+# bookHidden = false
+# Uncomment to exclude this page from the search
+# bookSearchExclude = true
+---
+
+# Azure
+
+Azure is Microsoft's cloud platform, through which Turing members have access to a wide range of computing and data storage resources.
+
+## Setting up Azure
+
+You can access Azure through the [Azure portal](https://portal.azure.com), using your Office 365 credentials.
+
+To experiment with Azure you should request a trial account with £300 credit using this [TopDesk Link](https://turingcomplete.topdesk.net/tas/public/ssp/content/serviceflow?unid=ac51b39d8bfc46f9bf41132ef8601b5e&from=7edfe644-ac0d-4895-af98-acd425ee0b19&openedFromService=true). When using Azure for projects you can use the same form to request a project-specific subscription with its own budget.
+
+## Quick links
+
+- [Azure Quickstart: Make a new VM using the command line]({{< relref "docs/technical_practices/azure/azure_make_vm_with_cli.md" >}})
+- [Azure Howtos](https://github.com/alan-turing-institute/howtos/tree/master/azure)
+
+## Managing your Azure account
+
+- [Using Microsoft Azure at the Turing](https://mathison.turing.ac.uk/page/2433)
+- [Monitor available credit on an Azure subscription](https://rcp-api-prod.azurewebsites.net/)
```

Comment:
The URL for RCTab has changed. This is the updated URL, which will not change again in the foreseeable future:

```suggestion
- [Monitor available credit on an Azure subscription](https://rctab-turing-prod.azurewebsites.net/)
```